README.md

```diff
@@ -6,60 +6,42 @@ docker/cli
 This repository is the home of the cli used in the Docker CE and
 Docker EE products.
 
-It's composed of 3 main folders
-
-* `/cli` - all the commands code.
-* `/cmd/docker` - the entrypoint of the cli, aka the main.
-
 Development
 ===========
 
-### Build locally
+`docker/cli` is developed using Docker.
 
-```
-$ make build
-```
+Build a linux binary:
 
 ```
-$ make clean
+$ make -f docker.Makefile binary
```

Comment:
Since this is blocking quite a few things, I'm fine with merging it as it is.

But I'm afraid we'll end up breaking people relying on the cli downstream if we don't update quickly afterwards.

`make` is the standard, but I agree it is ill fitted to deal with output depending on docker images (depending on the `Dockerfile`s themselves would be an improvement though).